staking_deposit/deposit.py

```diff
@@ -29,6 +30,18 @@ def check_python_version() -> None:
         sys.exit()
 
 
+def check_connectivity() -> None:
+    '''
+    Checks if there is an internet connection and warns the user if so.
+    '''
+    try:
+        socket.setdefaulttimeout(2)
+        socket.getaddrinfo('icann.org', 80)
+        click.pause(load_text(['connectivity_warning']))
+    except:  # noqa: E722
```

Comment:
This should only catch OSError since all socket exceptions [are subclasses of OSError](https://docs.python.org/3/library/socket.html#exceptions) and that should enable us to catch anything raised by socket calls here. It will also enable us to remove the `noqa: E722`.

Reply:
ack